src/lib/services/flows/service/flows.ts

```diff
@@ -56,3 +56,32 @@ export const remove = function (flow_id, options?, sdkOptions?) {
     payload?: never;
   }
 >;
+
+/**
+ * Run an instance of a particular Flow.
+ * @see https://globusonline.github.io/globus-flows/#tag/Runs/paths/~1flows~1%7Bflow_id%7D~1run/post
+ */
+export const run = function (flow_id, options?, sdkOptions?) {
+  return serviceRequest(
+    {
+      service: ID,
+      scope: SCOPES.VIEW_FLOWS,
+      path: `/flows/${flow_id}/run`,
+      method: HTTP_METHODS.POST,
+    },
+    options,
+    sdkOptions,
+  );
+} satisfies ServiceMethodDynamicSegments<
+  string,
+  {
+    query?: never;
+    payload: {
+      label?: string;
+      body: Record<string, any>;
+      tags?: string[];
+    };
+  }
+>;
+
+export const invoke = run;
```

Comment:
Is `invoke` the verb we want to use? Looks like the Python SDK uses `run_flow`. I'm not super invested in any terminology, just want us to be semi-consistent.

Reply:
I added `flows.invoke` as an alias based on the [documentation](https://globusonline.github.io/globus-flows/#tag/Runs/paths/~1flows~1%7Bflow_id%7D~1run/post) (and even considered adding `runs.invoke()` based on the organization), but I think just letting the URL model define the structure is the right way to go, so I dropped it.